src/bookshelf/test/bkswt_api_SUITE.erl

```diff
@@ -432,6 +433,42 @@ put_object(Config) when is_list(Config) ->
     % Cleanup
     ?assertEqual(ok, mini_s3:delete_bucket(Bucket, S3Conf)).
 
+has_header(_Header, []) ->
+    false;
+has_header(Header, [{Name, _Value}|T]) ->
+    Needle = string:to_lower(Header),
```

Comment:
[nit] This is called for every list element (i.e. the second argument), but doesn't change -- but since this is a test file and this doesn't get called often, I suppose that's fine.